### ISSUE TITLE
Custom bookshelves or tags for organization

### ISSUE THREAD

**Issue by QueenNiamh** [btpf/Alexandria#22]
Does it fit within the design goals of the app to add a bookshelf or tagging feature, to increase ease of use for larger book collections? At the moment, there isn't really a way to filter books by anything other than title; I would love to be able to categorize my collection instead of everything being jumbled together on the library screen.

**Comment by btpf** [comment]
TLDR: No, but maybe in the future.

Tagging/bookshelves were not a part of the original intended design, but began to make sense especially for comicbook series and syncing server contexts.

Comicbooks are also admittedly not the best experience, and a much needed ground up rewrite of Alexandria would solve this issue. However, I have entered a busy work period and have limited time. So I would like to invest it into furthering the strengths this app already has.

I currently view the addition of bookshelves and tagging as progression toward something more full blown and currently out of my scope. 

I think organizing books into folders on your computer and setting Alexandria as the default application offers a great workflow in the meantime.

My views on regarding this issue can change in the future.

Thanks, I hope this helps.
